### ISSUE TITLE
C# Library import issues for GPUParticles3D/CPUParticles3D etc

### ISSUE THREAD

**Issue by levithepants** [godotengine/godot#100957]
### Tested versions

Discovered in Godot v4.3.stable.mono.official [77dcf97d8]
Other versions unknown.

### System information

Godot v4.3.stable.mono - Windows 10.0.19045 - Vulkan (Forward+) - integrated AMD Radeon(TM) Graphics (Advanced Micro Devices, Inc.; 31.0.21910.5) - AMD Ryzen 5 5500U with Radeon Graphics (12 Threads)

### Issue description

Greetings!
There seems to be a mismatch between documentation and actual design when it comes to GPUParticles3D and all related nodes. In the documentation it is listed as "GPUParticles3D", but in GodotSharp.xml (_root_/GodotSharp/Api/Release/GodotSharp.xml) it is listed as "GpuParticles3D".
Thus, when trying to use GPUParticles3D, one gets the error "The type or namespace name 'CPUParticles3D' could not be found (are you missing a using directive or an assembly reference?)" because GPUParticles3D doesn't actually exist, only GpuParticles3D exists.

Of course, newly armed with this knowledge I can easily continue my work, but this was a stumbling block for me and I'm sure it will be for many others unless the documentation matches the xml file, regardless of which side yields to the other. 

Tl:Dr:
Please ether change the _root_/GodotSharp/Api/Release/GodotSharp.xml to match with "GPUParticles3D" and "CPUParticles3D" etc or change the documentation and syntax highlighting to match the xml file's GpuParticles3D and CpuParticles3D.

So, it's not a Godot breaker, but it is a rough edge that I'd like to see patched up!
Cheers!

### Steps to reproduce

Add GPUParticles3D into your code, at any point, and compile for immediate results.

### Minimal reproduction project (MRP)

N/A
